File: 백준/Bronze/25600. Triathlon/Triathlon.py
max_score = 0

for _ in range(int(input())):
    a, d, g = map(int, input().split())
    plus = d + g
    score = (a * plus) * 2 if a == plus else a * plus
    max_score = max(max_score, score)

print(max_score)

File: 백준/Bronze/25600. Triathlon/Triathlon.js
const fs = require('fs');
const filePath = process.platform === 'linux' ? '/dev/stdin' : 'run/input.txt';
const input = fs.readFileSync(filePath).toString().trim().split('\n');

let max_score = 0;

for (points of input.slice(1)) {
  const [a, d, g] = points.split(' ').map(Number);
  const plus = d + g;
  const score = a === plus ? a * plus * 2 : a * plus;
  max_score = Math.max(max_score, score);
}

console.log(max_score);
